types/svg-baker-runtime/src/sprite.d.ts

```diff
@@ -0,0 +1,26 @@
+import SpriteSymbol from './symbol';
+import SpriteConfig from './sprite.config';
+
+export default class Sprite {
+    config: SpriteConfig;
+    symbols: SpriteSymbol[];
+
+    constructor(config: SpriteConfig);
```

Comment:
```suggestion
    constructor(config?: SpriteConfig);
```
it always defaults to default configuration:
https://github.com/JetBrains/svg-mixer/blob/68f4f77f780d64883e26719a882c9893888c7099/packages/svg-baker-runtime/src/sprite.js#L10
For some reason the module avoids using optional params defaults